.github/workflows/cilium-integration-tests.yaml

```diff
@@ -0,0 +1,187 @@
+name: Cilium Integration Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - master
+  issue_comment:
+    types:
+      - created
+env:
+  KIND_VERSION: v0.18.0
+  CILIUM_REPO_REF: master
+  CILIUM_CLI_TAG: latest
+
+jobs:
+  cilium-connectivity-tests:
+    timeout-minutes: 360
+    name: Cilium Connectivity Tests
+    if: |
+      (github.event_name == 'issue_comment' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/test-cilium-integration')) ||
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Prepare variables for pushes to master
+        if: github.event_name == 'push'
+        run: |
+          echo "PROXY_IMAGE=quay.io/cilium/cilium-envoy" >> $GITHUB_ENV
+          echo "PROXY_TAG=${{ github.sha }}" >> $GITHUB_ENV
+
+      - name: Prepare variables for PR
+        if: github.event_name == 'pull_request'
+        run: |
+          echo "PROXY_IMAGE=quay.io/cilium/cilium-envoy-dev" >> $GITHUB_ENV
+          echo "PROXY_TAG=${{ github.event.pull_request.head.sha }}" >> $GITHUB_ENV
+
+      - name: Prepare variables for issue comment on a PR
+        if: github.event_name == 'issue_comment'
+        run: |
+          echo "PROXY_IMAGE=quay.io/cilium/cilium-envoy-dev" >> $GITHUB_ENV
+          echo "PROXY_TAG=${{ github.event.issue.pull_request.head.sha }}" >> $GITHUB_ENV
```

Comment:
This fails on an issue comment:
```
  echo "PROXY_IMAGE=quay.io/cilium/cilium-envoy-dev" >> $GITHUB_ENV
  echo "PROXY_TAG=" >> $GITHUB_ENV
```
See the empty `PROXY_TAG`. As seen on: https://github.com/cilium/proxy/actions/runs/4624591204/jobs/8179582498

The `Wait for Cilium proxy image to be available` step never completes because of this.

Reply:
yeah, seems like `${{ github.event.issue.pull_request.head.sha }}` is not set in case of a comment :man_facepalming:

Reply:
addressed by https://github.com/cilium/proxy/pull/160